### PR TITLE
Added docker labels - follows label schema namespace

### PR DESCRIPTION
**Pull request by waseem18**
**This PR adds labels to docker file. The changes follow the label schema namespace.** 

**Addresses the issue discussed at https://github.com/openebs/openebs/issues/446#issuecomment-333374139**:

@vharsh Let me know your review. 

How to run : `docker build --build-arg VERSION=0.4 --build-arg BUILD_DATE=2017-08-01T23:20:50.52Z .`

### CODE REVIEW

File: buildscripts/docker/Dockerfile [hunk 1]
```diff
@@ -6,5 +6,12 @@ FROM ubuntu:16.04
 
 COPY maya /usr/local/bin
 
-ENTRYPOINT ["maya"]
+ARG VERSION
+ARG BUILD_DATE
+
+RUN echo $BUILD_DATE
```
Comment (kmova) [inline buildscripts/docker/Dockerfile]: Why is this echo required?

Comment (waseem18) [reply buildscripts/docker/Dockerfile]: Oh! Sorry for that. I was just testing stuff and it got in. Will remove it

**Comment by kmova** [comment]
@waseem18 -- please help me with following queries:
- what will be the impact if the build args are not provided to the docker build command? 
- Can you provide the output of the command -- *docker inspect <image-id>*, on image created with and without passing the build args.?

**Comment by waseem18** [comment]
@kmova When the build args are not provided, the build fails with the error `One or more build-args [VERSION] were not consumed, failing build.`

Will put the output of `docker inspect` command soon!

**Comment by kmova** [comment]
@waseem18 

Just looking at the labels.. version and build-date are good. 

Can we remove vendor altogether. "OpenEBS is an OpenSource Project".. 

Let us change it to the following:
```
org.label-schema.name="maya"
org.label-schema.description="CLI for OpenEBS"
org.label-schema.url="http://www.openebs.io/"
org.label-schema.vcs-url="https://github.com/openebs/maya"
org.label-schema.schema-version="1.0"
```

**Comment by waseem18** [comment]
Sure! I've updated my previous comment @kmova 

Kindly look at it

**Comment by waseem18** [comment]
Made the label changes @kmova

File: buildscripts/docker/Dockerfile [hunk 2]
```diff
@@ -6,5 +6,15 @@ FROM ubuntu:16.04
...
 
-ENTRYPOINT ["maya"]
+ARG VERSION
+ARG BUILD_DATE
+
+LABEL org.label-schema.name="maya"
+LABEL org.label-schema.description="CLI for OpenEBS"
+LABEL org.label-schema.url="http://www.openebs.io/"
+LABEL org.label-schema.vcs-url="https://github.com/openebs/maya"
+org.label-schema.schema-version="1.0"
```
Comment (vharsh) [inline buildscripts/docker/Dockerfile]: `LABEL` missing :)

Comment (waseem18) [reply buildscripts/docker/Dockerfile]: added @vharsh

**Comment by waseem18** [comment]
Thanks @kmova

**Comment by vharsh** [comment]
fixes openebs/openebs#446

**Pull request merged**